src/js/components/Select/__tests__/SelectUncontrolled-test.js

```diff
@@ -10,10 +13,20 @@ import { Grommet } from '../..';
 import { Select } from '..';
 
 describe('Select', () => {
+  window.scrollTo = jest.fn();
```

Comment:
Why is this addition needed?

Reply:
Because every time the select is clicked on  `window.scrollTo` is called and if it is not mocked it gives an error. Since the component is clicked on in almost every test the cleanest way to implement it was to mock the function for the entire file then restore it at the end of the file. 
<img width="992" alt="Screen Shot 2020-07-08 at 12 35 17 PM" src="https://user-images.githubusercontent.com/49867133/86945815-80f26800-c117-11ea-880a-670a348adf90.png">